nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/parameter-contexts/ui/parameter-context-listing/parameter-context-table/parameter-context-table.component.ts

```diff
@@ -157,6 +170,9 @@ export class ParameterContextTable {
                 case 'description':
                     retVal = this.nifiCommon.compareString(this.formatDescription(a), this.formatDescription(b));
                     break;
+                case 'bounded process group':
```

Comment:
The label `bounded` is not quite right, it could be `bound process groups`, but just using `process groups` sounds better.

Reply:
Sure, I renamed the column to `process group` and adjusted the naming so that there is no `bounded`.